extensions/fairifier-rdf-extension/dtls/fairifier/PostFairDataToFairDataPoint.java

```diff
@@ -117,7 +113,7 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws Servl
                 catalogMetadata.setUri(f.createIRI(fdp.getString("baseUri")));
                 catalogMetadata.setIssued(f.createLiteral(date));
                 catalogMetadata.setModified(f.createLiteral(date));
-                catalogString = MetadataUtils.getString(catalogMetadata, RDFFormat.TURTLE).replaceAll("\\<" + catalogMetadata.getUri() + "\\>","<>");
+                catalogString = MetadataUtils.getString(catalogMetadata, RDFFormat.TURTLE, false).replaceAll("\\<" + catalogMetadata.getUri() + "\\>","<>");
```

Comment:
(pinging @rajaram5) Not a FAIRifier comment, but for the metadata library: if users prepare their RDF string to post to an FDP, we should handle the correct subject for them.